README.md

```diff
@@ -28,8 +28,11 @@ See [high level diagram](architecture.png)
 1. Specify [forge credentials](#specify-forge-credentials).
 1. Copy `AppBundles\InventorBinFolder.props.template` to `AppBundles\InventorBinFolder.props`
 1. Replace the `PATH_TO_YOUR_INVENTOR_BIN` string in the `AppBundles\InventorBinFolder.props` file with your actual Inventor bin folder path, for example: `C:\Program Files\Autodesk\Inventor 2021\Bin`
+1. (Optional) Choose network configuration for your application. By default the polling is enabled as it offers easier way to setup and run the application. This is OK for locally run applications and debuggin. However
+   in production using the new callback option is highly recommended to conserve resources. In order to enable the callback option modify `Publisher` section of the appsettings.json. 
+   Change `"CompletionCheck"` value from `"UsePolling"` to `"UseCallback"` and set `"CallbackUrlBase"` url to your server URL or ngrok tunnel URL for locally run application.
```

Comment:
Here we say the options are 'UsePolling' and 'UseCallback', but in the appsettings files (both), we use just 'Polling' or 'Callback'.

Reply:
Guys decided to sabotage my documentation writing skills and renamed the values :D, fixing